src/fsharp/FSharp.Core/array.fs

```diff
@@ -57,7 +57,7 @@ namespace Microsoft.FSharp.Collections
         let create (count:int) (x:'T) =
             if count < 0 then invalidArg "count" (SR.GetString(SR.inputMustBeNonNegative))
             let array: 'T[] = Array.zeroCreateUnchecked count
-            for i = 0 to Operators.Checked.(-) count 1 do // use checked arithmetic here to satisfy FxCop
+            for i = 0 to Operators.Checked.(-) array.Length 1 do // use checked arithmetic here to satisfy FxCop
```

Comment:
One of the functions that has it, has a comment about it being for FXCOP (I don't know what that is)
I did test with and without it, and it made no difference on that function.  I could test all of them though to be sure.